### DISCUSSION TITLE
The plan ahead 🚀

### DISCUSSION THREAD

**Discussion by mihir-bombay-studio** [dear-digital/linter#95] [Announcements]
👋 Hey Team,

I hope you're all doing well! First off, I want to extend a huge thank you for your hard work and dedication as you continue to learn and grow within our training program.

🌟 **Exciting News Ahead!** 🌟

I'm thrilled to share that we are transitioning to a more hands-on, practical approach to learning. Going forward, you'll have the opportunity to contribute directly to any of our [company-backed open-source repositories](https://github.com/dear-digital/) I will also start some new repositories with tech stack that we require junior developers for where you can contribute.

### Why This Matters
1. 🌱 **Personal Growth**: This experience will significantly enhance your GitHub resume.
2. 🤝 **Teamwork**: Learn to collaborate in a team environment similar to how we operate in dear digital.
3. 💼 **Career Path**: Interested in joining us full-time? Your contributions can serve as a stepping stone.

### How it Works
- 📊 We're developing an evaluation matrix captured by a GitHub app that will automatically build a leaderboard.
- 🎯 Points will be assigned based on the quality and quantity, among other factors.
- 📈 This is a win-win situation; you enhance your GitHub profile while also making a meaningful impact on our projects.

### Next Steps
Stay tuned for more information on how to get started and details about the GitHub app that will track your contributions. I have a draft document about joining which I'm sharing [Joining.pdf](https://github.com/dear-digital/linter/files/13072042/Joining.pdf). You can read it and also give feedback on how you as a fresher or open source contributor feel about it. Any scrutiny is also welcome.

Thank you for being part of this exciting journey. Your contributions and feedback are invaluable to us. Please feel free to share any questions or thoughts you may have.

**Comment by kmalap05** [comment]
@mihir-bombay-studio Is there any specific task we need to complete this week?

**Reply by mihir-bombay-studio** [reply]
@kmalap05 Thanks for asking,

Update for everyone we'll probably move to another repository now. It will mostly be https://github.com/dear-digital/shopify-theme-skeleton this one. I'm still planning out the architecture but it should be done by the end of next week so on 13th nov we would resume. We will towards a more automated approach towards the points and leaderboard which I'll inform in the coming days. Now that you have some experience with github, flow and github issues can directly start contributing to the project by checking issues and creating bugfix or feature requests. You must have gone through this documentation by Shopify to start contributing https://shopify.dev/docs/themes. Try setting up development stores on Shopify updating themes, checking the file structure and try implementing examples from the documentation to get ready to contribute. Let me know if you have any doubts.

**Reply by kmalap05** [reply]
Sure, we will use this time to acquire practical experience with Shopify themes.

**Reply by mihir-bombay-studio** [reply]
https://github.com/mdn/curriculum If you are interested to dive deeper into frontend you can refer this curriculum.

**Reply by mihir-bombay-studio** [reply]
Happy Diwali everyone!

I've started the new repository, it's called [Shopify theme skeleton](https://github.com/dear-digital/shopify-theme-skeleton). I'm also adding issues to it which you can pick up. The point system for internship has changed and you can find the rules in the [rewards](https://github.com/dear-digital/shopify-theme-skeleton/blob/main/rewards.md) document. Points are reset and everyone starts at L0.

https://github.com/dear-digital/shopify-theme-skeleton

Make sure to go through all the documentation and the repository once before you get started. Good luck!

**Reply by tanujbordikar** [reply]
Happy Diwali Sir, the new repository looks extremely good and challenging, excited to contribute!👍

**Reply by anishdalvi** [reply]
Happy Diwali Sir!
Excited to contribute to this new repository of Shopify Skeleton Theme 🙌🏻
